core/storage/suggestion/gae_models.py

```diff
@@ -809,7 +809,7 @@ def get_in_review_translation_suggestions_by_exp_ids(
     @classmethod
     def get_in_review_question_suggestions_by_offset(
         cls,
-        limit: int,
+        limit: Optional[int],
```

Comment:
Hey @seanlip, I have made this parameter optional as per  Sagang's suggestion (https://github.com/oppia/oppia/pull/18664#discussion_r1274153430)

Reply:
I'm not really sure about this -- operations to the datastore should always be bounded. 

I guess you're making this optional to match the other methods in this file, right? But I'm confused about why they got marked optional in the first place. What's the rationale for making things consistent in this direction vs removing the optionality from the other methods? (In general I don't think letting limits be optional is a standard practice in our storage layer.)

Reply:
Hm, I'm not actually sure what the full original code looked like here, but thinking this over again, I agree that we shouldn't accept an optional limit for the question suggestion fetch methods since I think question suggestion fetches should always accept a limit. Sorry about that.

We're not passing in a limit to `get_reviewable_translation_suggestions_by_offset` though since for this sorting feature we have to fetch all the results from the datastore at once and sort the results client-side.

Reply:
Ah, ok. In that case, would it be possible to add detailed comments to the Optional[int] one, so that other devs know that it's not a pattern to follow?

/cc @Ash-2k3